typescript/sdk/src/gas/calculator.test.ts

```diff
@@ -58,53 +87,64 @@ describe('InterchainGasCalculator', () => {
     provider.clearMethodResolveValues();
   });
 
-  describe('estimatePaymentForHandleGasAmount', () => {
+  describe('estimatePaymentForGas', () => {
     it('estimates origin token payment from a specified destination gas amount', async () => {
       // Set destination gas price to 10 wei
       provider.setMethodResolveValue(
         'getGasPrice',
         BigNumber.from(SUGGESTED_GAS_PRICE),
       );
 
-      // Stub the checkpoint relay gas cost
-      sinon
-        .stub(calculator, 'checkpointRelayGas')
-        .returns(Promise.resolve(BigNumber.from(CHECKPOINT_RELAY_GAS)));
+      const estimatedPayment = await calculator.estimatePaymentForGas(
+        origin,
+        destination,
+        BigNumber.from(HANDLE_GAS),
+      );
+
+      // 100k gas * 10 gas price * ($5 per origin token / $10 per origin token)
```

Comment:
I think this is originally my fault lol, this should be "$5 per destination token" here and elsewhere in this file

Reply:
Good catch!